vendor/brave-ios/Ads/BATBraveAds.h

```diff
@@ -131,6 +131,12 @@ NS_SWIFT_NAME(BraveAds)
 - (void)reportAdNotificationEvent:(NSString *)notificationUuid
                         eventType:(BATAdNotificationEventType)eventType;
 
+/// Update ad totals on month roll over, optionally reconcile with server
+- (void)updateAdRewards:(BOOL)shouldReconcile;
+
+/// Get the number of ads received and the estimated earnings of viewing said ads for this cycle
+- (void)detailsForCurrentCycle:(void (^)(NSInteger adsReceived, double estimatedEarnings, NSDate *nextPaymentDate))completion;
```

Comment:
Add `NS_SWIFT_NAME(adsDetailsForCurrentCycle(_:))` to end (See removed function declaration in `BATBraveLedger.h`)

This change will update Swift call signature and require and update on brave-ios branch

Reply:
Fixed